Packs/Uptycs/ReleaseNotes/1_0_9.md

```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### Uptycs
+- updated Uptycs jwt token authentication and added new commands.
```

Comment:
```suggestion
- updated Uptycs jwt token authentication and added new commands.
- Updated the Docker image to: *demisto/uptycs:1.0.0.63766*.
```
Please also add a line on the updated docker image.

Reply:
updated